utils/test/RunningTests.md

```diff
@@ -0,0 +1,67 @@
+#Running tests for solutions-geoprocessing-toolbox
+
+##Sections
+* [Introduction](#introduction)
+* [Requirements](#requirements)
+	* [For ArcGIS Pro](#for-arcgis-pro)
+	* [For ArcGIS Desktop](#for-arcgis-desktop)
+* [Running the tests](#running-the-tests)
+* [Log files](#log-files)
+* [Reporting errors as Issues](#reporting-errors-as-issues)
+
+##Introduction
+The unittests in this section of the repository are designed to test the geoprocessing tools contained within to make sure they are running consistently under different conditions. Automated testing, unittests are one type, help reduce the amount of time necessary to find defects or confirm their success. An automated test can run all of the tools in a significantly shorter amount of time than several people testing the tools manually. This should not be taken as an opportunity to skip other types of testing (ad-hoc, defect verification, template testing, system testing, etc.).
+
+##Requirements
+###For ArcGIS Pro
+* ArcGIS Pro 1.1+
+* Python 3.4+
+
+###For ArcGIS Desktkop
+* ArcGIS for Desktop 10.3.1+
+* Python 2.7+
+
+##Running the tests
+The goal is to run the test suites for all of the tools against different versions of operating systems and versions of Python/ArcGIS.
+
+1. Check your Python versions. You need to have at least ONE of two required versions listed above.
+2. Modify **TestKickStarter.bat** to run tests for the versions you have installed.
```

Comment:
Actually called TestKickStart~~er~~.bat - you could also directly link to this file `[file](./TestKickStart.bat)` so reader could easily link/see